capabilities/promtail/zarf.yaml

```diff
@@ -1,11 +1,27 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
 kind: ZarfPackageConfig
 metadata:
-  name: example
-  description: "Example description"
+  name: uds-capability-promtail
+  description: "UDS Promtail Capability"
+  url: "https://grafana.com/docs/loki/latest/"
+
 
 components:
-  - name: test
-    actions:
-      onCreate:
-        after:
-          - cmd: echo "What can Doug do for you?"
+  - name: promtail
+    required: true
+    description: "Deploy Promtail"
+    charts:
+      - name: promtail
+        url: https://grafana.github.io/helm-charts/
+        version: 6.15.3
+        namespace: promtail
+        gitPath: charts/promtail
+        valuesFiles:
+          - values/values.yaml
+      - name: uds-promtail-config
```

Comment:
same comment regarding app vs config as their own components. maybe it's fine--but we need to decided and try to stick to a common pattern if we can I think